tests/openssl/openssl_sha.cpp

```diff
@@ -112,6 +112,7 @@ static int ssl_sha_init(struct test* test)
         return EXIT_SUCCESS;
     }
     else {
+        log_skip(ResourceIssueSkipCategory, "s_EVP_DigestInit_ex/s_EVP_DigestUpdate/s_EVP_DigestFinal_ex/s_EVP_get_digestbyname one or more of these have failed");
```

Comment:
Do we get an error string from OpenSSL? @AthenasJimenez

Reply:
No, we don't. But this skip only happen if Sandstone could not load openssl functions. So for me the skip reason should look like:
```
log_skip(ResourceIssueSkipCategory, "Openssl library is not available or the current version is not supported")
```

Reply:
Ack. Please spell "OpenSSL".

Reply:
Done